genai-on-vertex-ai/vertex_ai_search/ingesting_unstructured_documents_with_metadata.ipynb

```diff
@@ -182,7 +182,7 @@
     },
```

Comment:
<p>nit: include space in the header  <code>## Delete the Search App</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>Thanks Done</p>